setup.py

```diff
@@ -4,7 +4,7 @@
 
 setup(
     name = 'pylutron',
-    version = 'TEMPLATE_VERSION',
+    version = '31',
```

Comment:
Please drop this change.  It should not be needed.

Reply:
understood